packages/react-compose/src/next/types.tsx

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Two things:
1. This should be a .ts file.
2. You probably want to import `React` here.

Reply:
awesome catch